src/fusor/translator.py

```diff
@@ -127,9 +127,13 @@ def _get_gene_element_unnormalized(self, symbol: str) -> GeneElement:
         """
         return GeneElement(
             gene={
-                "id": f"gene:{symbol}",
-                "label": symbol,
-                "type": "Gene",
+                "primaryCoding": {
+                    "id": f"gene:{symbol}",
+                    "code": f"gene:{symbol}",
+                    "system": f"Naming system for {symbol} unavailable",
```

Comment:
Did you check with Alex on this?

Reply:
Do you have to use primaryCoding? Could you just use `id`?

Reply:
I haven't checked yet with Alex, but I modified to remove the `primaryCoding` attribute and report only the `name` and `conceptType`.